### ISSUE TITLE
UUIDs not shown

### ISSUE THREAD

**Issue by iikku** [speed47/btrfs-list#21]
UUIDs are not shown.

```
└> sudo btrfs filesystem show
Label: 'label'  uuid: bd56f0a1-b50b-49a3-846b-f3f9b6fbe5a3
        Total devices 1 FS bytes used 2.53TiB
        devid    1 size 3.64TiB used 2.57TiB path /dev/sdc
```

versus

```
└> sudo ./btrfs-list -q -a 
NAME      ID PARENT TOPLVL GEN CGEN       UUID PARENTUUID  RCVD_UUID                OTIME    TYPE     EXCL  MOUNTPOINT
label    -      -      -   -    -          -          -          -                    -      fs    2.53T (single/dup, 1.10T/3.64T free, 30.27%)
    [main]  5      -      -   -    -          -          -          -                    - mainvol       -  /media/user/label
```

**Comment by speed47** [comment]
I can't reproduce the problem on my side with just the info you gave, I do have the UUIDs on my output with the command you're typing. Maybe this has to do with the `btrfs-progs` version, which one are you using? This can be seen with:
```
btrfs-list --version

**Comment by iikku** [comment]
`btrfs-list --version` prints
`btrfs-list v2.2 using btrfs v6.6.3`

`apt show btrfs-progs` prints
```
Package: btrfs-progs
Version: 6.6.3-1.1build2
```

**Comment by speed47** [comment]
I can reproduce it with btrfs-progs v6.1, which was broken for uuids, as seen by the fixing commit https://github.com/kdave/btrfs-progs/commit/b7e3acd615529530627c402da8f98192f1bb57f3
Testing with all released versions to see if more versions are affected.

**Comment by speed47** [comment]
Tested all the released versions upstream, v6.1 seems the only one to have the problem, at least that I can reproduce.
Can you try to launch manually:
```
sudo btrfs sub list -us /media/user/label
```
If you don't see uuids there, it means that your version is somehow affected too (6.6.3-1.1build2, unsure if you compiled it yourself or if it's from a distrib)

**Comment by iikku** [comment]
So... pebkac

I didn't have any subvolumes, which is probably why subvolume uuids weren't shown. I created one and its shown:

```
NAME         ID PARENT TOPLVL    GEN   CGEN       UUID PARENTUUID  RCVD_UUID                OTIME    TYPE     EXCL  MOUNTPOINT
nelonen       -      -      -      -      -          -          -          -                    -      fs    2.47T (single/raid1, 1.62T/4.09T free, 39.52%)
    [main]    5      -      -      -      -          -          -          -                    - mainvol       -  /media/iikku/nelonen
    example 6114      5      5 151392 151392 0f1e..0c8a          -          -                    -  subvol       -  
```

However, this points out an indentation issue, the parentuuid dash (-) on the last row is one space too far right.

Also I wonder if the original the original issue is more than a misunderstanding. `btrfs filesystem show /media/user/label` does show the label uuid, which isn't shown on the btrfs-list subvolume uuid list. Is this intended?

**Comment by speed47** [comment]
About the indentation issue, can you try the latest (unreleased) version from the main branch? I fixed it yesterday (#19)  but wanted to wait for this issue to be closed/solved before formally releasing a new version. If you can confirm that the current main branch fixes the indentation issue you have, that would be good!

About the UUID of the main filesystem, I suppose I could add it in the UUID column if present indeed, even if the column was intended for subvolumes UUID at first, I suppose it doesn't hurt (parentuuid and rcvd_uuid will obviously stay empty however)

**Comment by iikku** [comment]
I checked out the latest version and the indentation is fixed, great!
